content/workshops/containers_101/exercise1.3.adoc

```diff
@@ -0,0 +1,104 @@
+---
+title: "Exercise 1.3 - Working with the Dockerfile"
+workshops: containers_101
+workshop_weight: 50
+layout: lab
+---
+
+:badges:
+:icons: font
+:imagesdir: /workshops/containers_101/images
+:source-highlighter: highlight.js
+:source-language: yaml
+
+
+= Exercise 1.3 - Working with the Dockerfile
+
+Once you understand how images and containers can be created from the command line, you can try building containers in a more permanent way. Building container images from Dockerfile files is by far the preferred way to create Docker formatted containers, as compared to modifying running containers and committing them to images.
+
+The procedure here involves creating a Dockerfile file that includes many of the features illustrated earlier:
+
+- Choosing a base image (i.e. Fedora)
+
+- Installing the packages needed (i.e. an Postgresql Server)
+
+- Mapping the server’s port (i.e. 5432)
+
+- Launching the Database server
+
+First get the Dockerfile from GitHub using curl.
+
+
+.Get file
+[source, bash]
+----
+curl -s -o Dockerfile https://raw.githubusercontent.com/jason-callaway/openstackdc/master/Dockerfile_fedora-postgres
```

Comment:
Not necessary for this to get published, but probably a good idea to move this file to something under RedHatGov.